client/app/scripts/utils/math-utils.js

```diff
@@ -18,3 +18,10 @@
 export function modulo(i, n) {
   return ((i % n) + n) % n;
 }
+
+// Does the same that the deprecated d3.round was doing.
+// Possibly imprecise: https://github.com/d3/d3/issues/210
+export function round(number, precision = 0) {
```

Comment:
The fact that it outputs a string with a fixed precision :) What we want to do here is to not show the trailing zeros, treating the `precision` as maximal precision instead of fixed. So for example, `2.34.toFixed(4) === "2.3400"` while `formatFlexiblePrecision(2.34) === "2.34"`. On the other hand `11.01234.toFixed(4) === formatFlexiblePrecision(11.01234) === "11.0123"`. It seems at first that this could be achieved by `Number.prototype.toPrecision` though, but that one's not good either as it starts rounding up more than just the decimal part when the values get very big.